rest_api/openapi.yaml

```diff
@@ -385,6 +385,76 @@ paths:
         503:
           $ref: "#/responses/503ServiceUnavailable"
 
+  /receipts:
+    get:
+      summary: Fetches the receipts for a set of transactions
+      description: |
+        Fetches an array of objects for each receipt requested.
+
+        The receipt(s) you want to retrieve can be specified using the `id`
+        filter parameter, where `id` refers to the transaction id of the
+        transaction the receipt is associated with.
+      parameters:
+        - name: id
+          in: query
+          description: A comma-separated list of transaction ids
+          type: string
+          required: true
+      responses:
+        200:
+          description: Successfully retrieved transactions
+          schema:
+            properties:
+              data:
+                type: array
+                items:
+                  - $ref: "#/definitions/TransactionReceipt"
+              link:
+                $ref: "#/definitions/Link"
+        400:
+          $ref: "#/responses/400BadRequest"
+        500:
+          $ref: "#/responses/500ServerError"
+        503:
+          $ref: "#/responses/503ServiceUnavailable"
+
+    post:
+      summary: Fetches the committed statuses for a set of batches
+      description: |
+        Identical to `GET /batch_status`, but takes ids of batches as a JSON
+        formatted POST body rather than a query parameter. This allows for many
+        more batches to be checked and should be used for more than 15 ids.
```

Comment:
Should read:
```
Identical to `GET /receipts`, but takes ids of transactions as a JSON
formatted POST body rather than a query parameter. This allows for many
more receipts to be fetched and should be used with more than 15 ids.
```